components/button/test/button-mixin.test.js

```diff
@@ -0,0 +1,47 @@
+import { defineCE, expect, fixture, oneEvent } from '@open-wc/testing';
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { ButtonMixin } from '../button-mixin.js';
+
+const tagName = defineCE(
+	class extends ButtonMixin(LitElement) {
+		render() {
+			return html`
+				<button>Test Button</button>
+			`;
+		}
+	}
+);
```

Comment:
This is pretty sweet -- we can define elements inline so we hopefully don't need as many test/demo components.